Kudu.Core/Deployment/Generator/SiteBuilderFactory.cs

```diff
@@ -194,6 +194,17 @@ private ISiteBuilder ResolveProject(string repositoryRoot, string targetPath, ID
             {
                 return DetermineProject(repositoryRoot, projects[0], perDeploymentSettings, fileFinder);
             }
+            
+            // Check for ASP.NET 5 project without VS solution or project
+            string projectJson;
+            if (AspNet5Helper.TryAspNet5Project(targetPath, out projectJson))
+            {
+                return new AspNet5Builder(_environment,
+                                           perDeploymentSettings,
+                                           _propertyProvider,
+                                           repositoryRoot,
+                                           projectJson);
+            }
 
```

Comment:
@ahmelsayed do you see any regression by doing TryAspNet5Project before WebSiteProject?   In case, they are not mutually exclusive.